code/src/components/MovieListed.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export const MovieListed = ({ id, title, poster_path }) => {
+  return (
+    <Link className="movie-card-link" to={`/movies/${id}`}>
+      {/* "to" decides the path name and the value of the movieId variable I have in App.js and MovieDetails */}
+      <img className="movie-card-image" src={`https://image.tmdb.org/t/p/w500${poster_path}`} alt={title}></img>
+    </Link >
+  )
+}
```

Comment:
It's so interesting to see how many different ways you could complete this component. You could basically pass it whatever you like! I also passed a `release_date` to the component, what you did is more than enough!